explainers/update-rendering.md

```diff
@@ -0,0 +1,44 @@
+## `updateRendering()`
+
+### TL;DR
+updateRendering() is an asynchronous JavaScript function on Element which
+requests that the rendering for the element be updated in preparation for
+display.
+
+### Motivation
+The web includes a number of features and heuristics (such as
+content-visibility, containment and others) that allow the user-agent to skip
+rendering work for elements and contents of elements. This is done with the
+intent to allow other content, animations and interactions to remain smooth and
+get as much CPU time as possible. However, there are situations where this
+neglects the site intent of showing currently skipped content shortly. In other
+words, if the website intends to show an element whose contents are currently
+skipped, then skipping work may cause jank when the contents are ultimately
+presented.
+
+### Proposal
+The proposal is to add the following function, which lets the user-agent know
+that the element’s contents should be updated in preparation for display:
+
+Signature: `Promise Element.updateRendering(Priority?)`
+
+The function takes a Priority as an argument, with the following proposed values
```

Comment:
These values should conform to the postTask API. https://wicg.github.io/scheduling-apis/#sec-task-priorities